linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_syslog_plugin_activated/rule.yml

```diff
@@ -45,6 +45,7 @@ references:
     cobit5: APO11.04,APO12.06,BAI03.05,BAI08.02,DSS02.02,DSS02.04,DSS02.07,DSS03.01,DSS05.04,DSS05.07,MEA02.01
     iso27001-2013: A.12.4.1,A.12.4.2,A.12.4.3,A.12.4.4,A.12.7.1,A.16.1.4,A.16.1.5,A.16.1.7
     cis-csc: 1,11,12,13,14,15,16,19,3,4,5,6,7,8
+    srg: SRG-OS-000051-GPOS-00024
```

Comment:
OK - so where would this one fit?  It is not the default setting on RHEL, but should be mapped somewhere since it configured audit to use the local syslog server?  Maybe SRG-OS-000480-GPOS-00227?

Reply:
updated to ``srg: SRG-OS-000479-GPOS-00224,SRG-OS-000342-GPOS-00133``